### DISCUSSION TITLE
cannot edit alpha value of basic material when a texture is plugged in

### DISCUSSION THREAD

**Discussion by MarcVdH** [cables-gl/cables_docs#601] [! Welcome - readme first!]
before the last update I was able to control the transparency of a basic material when an image or text was plugged in straight in the material op. now i need an extra color value op to access the Alpha value.    
![image](https://github.com/cables-gl/cables_docs/assets/126583906/2ea909ef-8f27-4b99-9566-596cb3c10ff2)
![image](https://github.com/cables-gl/cables_docs/assets/126583906/5e2a5bb7-ccab-406c-92e7-8ae74d76d1ac)

**Comment by pandrr** [comment]
ouch thank you! fixed on dev.cables.gl will be moved to live soonish

**Reply by MarcVdH** [reply]
awesome, alpha is directly accessible again. that was fast!
